docs/manuals/source/Appendix/Howtos.rst

```diff
@@ -945,6 +945,144 @@ A very simple corresponding shell script (:command:`bpipe-restore.sh`) to the me
    cat - > /tmp/dump.sql
    exit 0
 
+
+.. _section-MigrationMysqlToPostgresql:
+
+Migrate |mysql| to |postgresql|
+---------------------------
+
+:index:`\ <single: Migration; MySQL; postgreSQL; bareos-dbcopy; dbcopy>`
```

Comment:
```suggestion
:index:`\ <single: Migration; MySQL; postgreSQL; bareos-dbcopy; dbcopy>`
.. index::
   single: Migration from MySQL to PostgreSQL
   single: Database; Migration; MySQL
   single: Database; Migration; PostgreSQL
   pair: Migration; MySQL
   pair: Migration; PostgreSQL
   single: bareos-dbcopy
   see: dbcopy; bareos-dbcopy
```

The semicolon creates an index hierarchy. However, HTML output supports only 2 (3) hierarchy levels.
I assume, my suggestion matches your intention better.

Reply:
Good suggestion. I already changed that but your's even better.